src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/UserRoleOptions.tsx

```diff
@@ -0,0 +1,36 @@
+import { defineMessages, IntlShape } from 'react-intl';
+
+import { PlaylistRole } from '../types/playlistAccess';
+
+const messages = defineMessages({
+  adminLabel: {
+    defaultMessage: 'Administrator',
+    description: 'Administrator role label in playlist access select.',
+    id: 'features.Playlist.features.UpdatePlaylist.components.UserRoleOptions.adminLabel',
+  },
+  instructorLabel: {
+    defaultMessage: 'Instructor',
+    description: 'Instructor role label in playlist access select.',
+    id: 'features.Playlist.features.UpdatePlaylist.components.UserRoleOptions.instructorLabel',
+  },
+  studentLabel: {
+    defaultMessage: 'Student',
+    description: 'Student role label in playlist access select.',
+    id: 'features.Playlist.features.UpdatePlaylist.components.UserRoleOptions.studentLabel',
+  },
+});
+
+export const userRoleOptions = (intl: IntlShape) => [
+  {
+    label: intl.formatMessage(messages.adminLabel),
+    key: PlaylistRole.ADMINISTRATOR,
+  },
+  {
+    label: intl.formatMessage(messages.studentLabel),
+    key: PlaylistRole.STUDENT,
+  },
```

Comment:
should be moved after INSTRUCTOR IMO, to have this list `admin`, `instructor` and `student`